test/playwright/visual-regression/pages/homepage.spec.ts

```diff
@@ -35,6 +48,15 @@ for (const dir of languageDirections) {
       })
     )
 
+    breakpoints.describeEvery(({ expectSnapshot }) =>
+      test(`${dir} full page redesigned`, async ({ page }) => {
```

Comment:
If we add something like "old design" to the other test, it will be easier to clean up after the full transition to the new designs.

Reply:
Makes sense, updated test names.